cxx-checks/src/main/java/org/sonar/cxx/checks/StringLiteralDuplicatedCheck.java

```diff
@@ -67,7 +67,7 @@ public void visitFile(AstNode node) {
 
   @Override
   public void visitNode(AstNode node) {
-    if (node.is(CxxGrammarImpl.LITERAL)) {
+    if (node.is(CxxGrammarImpl.LITERAL) && !node.getToken().isGeneratedCode()) {
```

Comment:
@tomassdepakin think because of

```Java
public void init() {
    subscribeTo(CxxGrammarImpl.LITERAL);
  }
```
we don't need `if (node.is(CxxGrammarImpl.LITERAL)`. maybe you can remove it and try without again. Other changes are ok.